make/include/contrib.defs

```diff
@@ -94,7 +94,7 @@ define import.CONTRIB.defs
 
     $(1).CONFIGURE.args.dir    = cd $$(1);
     $(1).CONFIGURE.args.host   = --host=$$(1)
-    $(1).CONFIGURE.args.build  = --build=$$(1)
+    $(1).CONFIGURE.args.build  = --build=x86_64-unknown-linux-gnu
```

Comment:
Need some help with make/include/contrib.defs , for now it is hard coded as "x86_64-unknown-linux-gnu" (msys gives "x86_64-pc-msys" which cause problems with some contributes and ./configure --host=x86_64-w64-mingw32 --build=x86_64-pc-msys ... )